index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+    <header>
+    <nav>
+      <a class="logo"href="#homa">Emmabgra</a>
+      <ul class="navbar__ul">
+        <li class="navrbar__link"><a href="#portfolio">Portfolio</a></li>
+        <li class="navrbar__link"><a href="#about">About</a></li>
+        <li class="navrbar__link"><a href="#contact">Contact</a></li>
+      </ul>
+      <img class="menu" src="img/Union.svg" alt="">
+    </nav>
+
+        <div class="description">
+      <h1 class="header">I’m Emmanuel. <br>
+        Glad to see you!</h1>
+      <p class="parg">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      
+      <p class="let__go">LET’S CONNECT</p>
+      
+      <ul class="social__media">
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/twitter (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/Linkedin icon (1).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/medium (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/github (2).svg" alt=""></a></li>
+        <li class="social__media__link"><a href="#twitter"><img class="social__media__img" src="img/angellist.svg" alt=""></a></li>  
+      </ul>
+    </div>  
+      <img class="backg__image" src="img/Header bg.svg" alt="">
+      <img class="backg__image__phone" src="img/header-shapes mobile@2x.svg" alt="">
+  </header>
```

Comment:
- [ ] The container of the headline section should be a `section` tag. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. This means that your `<header>` should contain only the logo and the `nav` hamburger menu element. The the `<div class="description">`  should be a section tag. Like so 👇🏾 


```html
<header>
   <logo>
   <nav>
</header>
<section>
   <h1>
    other elements.
</section>
```